OneCommunityGlobal/HGNRest
# Parthj unit test time off request controller

## Description
This PR contains unit tests for `timeOffRequest Controller`

## Related PRS (if any):
None. 

## Main changes explained:
- Adds unit test code for `timeOffRequest Controller`
- Adds requirement files (with `.md` extension) highlighting the different test cases

## How to test:
1. check into current branch by `git checkout parthj_unit_test_timeOffRequestController`
2. do `npm install` 
3. execute test by `npm run test timeOffRequestController`

## Screenshots or videos of changes:

The result should look as follows:

<img width="1038" alt="Screenshot 2024-06-28 at 5 15 18 PM" src="https://github.com/OneCommunityGlobal/HGNRest/assets/56110372/235a37a5-de05-472a-b3ef-437d3378b9ae">
